sdk/include/platform/sunburst/platform-spi.hh

```diff
@@ -0,0 +1,227 @@
+#pragma once
+#include <cdefs.h>
+#include <debug.hh>
+#include <stdint.h>
+
+/**
+ * A Simple Driver for the Sonata's SPI.
+ *
+ * Documentation source can be found at:
+ * https://github.com/lowRISC/sonata-system/blob/1a59633d2515d4fe186a07d53e49ff95c18d9bbf/doc/ip/spi.md
+ *
+ * Rendered documentation is served from:
+ * https://lowrisc.org/sonata-system/doc/ip/spi.html
+ */
+struct SonataSpi
+{
+	/// Interrupt State Register
+	uint32_t interruptState;
+	/// Interrupt Enable Register
+	uint32_t interruptEnable;
```

Comment:
It looks as if we're missing the interrupt definitions in the board JSON?

Reply:
I checked whether the interrupts are actually hooked up in the RTL, and I haven't tested them if they are. I'm apprehensive about adding them to the board description file until I have.

Reply:
Might be nice to have a comment explaining this as a FIXME?

Reply:
Just checked the RTL; the SPI block currently doesn't have interrupts.

I have added a comment saying that these registers are reserved for future use.